### ISSUE TITLE
Stuck on "Beautification in progress" with HTML

### ISSUE THREAD

**Issue by end-user** [Glavin001/atom-beautify#1480]
When I CTRL+ALT+B, I get a spinner but no results.   It does work in other languages (or at least js).

![capture](https://cloud.githubusercontent.com/assets/4923193/22207997/4929f6b4-e14f-11e6-98ae-82718d257b6b.PNG)

Trivial HTML example:

```
<div class="modal-header">
<h3 class="modal-title">test</h3>
</div>
```

I expected

```
<div class="modal-header">
    <h3 class="modal-title">test</h3>
</div>
```

I have:
- [ ] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues

I can't see how to get any response and so there's no debug.  I've verified the same result with a full HTML document.

**Comment by lunchboxsuhi** [comment]
same issue for me, current version install is 0.29.16, windows 10.

**Comment by bjax13** [comment]
Getting the same issue 0.29.16 Mac Sierra

**Comment by prettydiff** [comment]
Do you guys have a space in your directory structure, such as `C:\Users\Austin Cheney\...`?  To me this sounds like a duplicate of these issues:

* https://github.com/Glavin001/atom-beautify/issues/1434
* https://github.com/Glavin001/atom-beautify/issues/1464

**Comment by bjax13** [comment]
Thanks for the tip.  I checked my directory structure and do not have any spaces that I can find. 

MacHD/Users/mylappytappy/devmtn/personalProjects/sandBox is the file path to the project that I am working on.   I checked each of the file names in the directory and non have spaces.    Although I will admit i did not check every file name in my node_modules folder.

**Comment by ubaid-rehman** [comment]
Same issue here, after update to 0.29.16

**Comment by jamesjryan** [comment]
Also experiencing this.

**Comment by Glavin001** [comment]
I wonder if this change will fix it! https://github.com/Glavin001/atom-beautify/pull/1432

**Comment by Glavin001** [comment]
Duplicate of #1432.

Published fix of #1432 to v0.29.17! Thank you for your patience!

**Comment by ubaid-rehman** [comment]
Thanks a lot. v0.29.17 fixed it.
